plugins.d/alarm-notify.sh

```diff
@@ -535,7 +560,7 @@ send_pushover() {
 # pushbullet sender
 
 send_pushbullet() {
-    local userapikey="${1}" recipients="${2}"  title="${3}" message="${4}" httpcode sent=0 user
+    local usersid="${1}" recipients="${2}"  title="${3}" message="${4}" httpcode sent=0 user
```

Comment:
It doesn't =/, It's a bug in my Bash for windows 10. I had a ssh shell crash when I was editing it in vi, then when I reopened I recovered the file, but for some reason vi would randomly open the older corrupted one in swap. I changed it on accident and pushed it, fixed it and pushed it again. While changing something else bash must have opened my older corrupted file and that's how it ended back into the commit. I'll fix it in an hour after I eat.